### DISCUSSION TITLE
Add light mode

### DISCUSSION THREAD

**Discussion by ghost** [pkgxdev/pkgx#792] [Ideas]
Please add light mode to your website/docs.

Thank you. 🙏

**Comment by gdmnkam** [comment]
Totally agree!

**Comment by kishaningithub** [comment]
Wondering what makes you wish for light mode. Can you kindly elaborate?

**Reply by ghost** [reply]
My visual impairment. Reading bright text on a dark background is hard for me.

**Comment by mxcl** [comment]
I have added the light/dark toggle to our docs because it is gitbook. Sadly I cannot offer a timeline on light mode support for the rest of our properties since, as a startup with limited resources, we just don't have the bandwidth.

I know there are browser extensions that can do this automatically, and while they probably have issues, is that not a possible solution?

**Reply by ghost** [reply]
Thank you very much for your quick response! This is so much better.
--
So far I did only find extensions for dark mode all the things. I would love to see people in the web dev community thinking "light mode first". Thanks for caring! 🙏

**Comment by gdmnkam** [comment]
Nice job guys! Thank you!
